src/content/staking/solo/index.md

```diff
@@ -25,7 +27,7 @@ A solo staker receives rewards directly from the protocol for keeping their vali
 
 ## Why stake solo? {#why-stake-solo}
 
-Solo staking comes with more responsibility, but provides you with maximum control over your funds and staking setup.
+Solo staking comes with more responsibility - this is why validators get paid in ETH - but provides you with maximum control over your funds and staking setup.
```

Comment:
Don't all stakers get paid in ETH? This seems to imply otherwise.

Reply:
yes they do - i was trying to convey the point that the reason there is a return is because you perform some useful function and have some responsibility --> not free money.

Reply:
But this is true of all forms of staking. Not strongly opposed, but I don't see the utility of this statement. 

If someone is looking at our solo staking page, I'd guess they are likely to know that staking is taking on some economic responsibility for returns.

Reply:
```suggestion
Solo staking comes with more responsibility but provides you with maximum control over your funds and staking setup.
```